elements.less

```diff
@@ -69,6 +69,17 @@
 
 
 
+/* Animation Mixins
+ * ============================================================ */
+.animation-duration(...) {
+  -webkit-animation-duration: @arguments;
+	 -moz-animation-duration: @arguments;
+	   -o-animation-duration: @arguments;
+		  animation-duration: @arguments;
+}//.animation-duration 
```

Comment:
Have replaced with your snippet. Can you clarify what settings should be used? Thanks